packages/opentelemetry-resources/test/detectors/browser/BrowserDetector.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as sinon from 'sinon';
+import { Resource } from '../../../src';
+import { browserDetector } from '../../../src/detectors/BrowserDetector';
+import {
+  assertResource,
+  assertEmptyResource,
+} from '../../util/resource-assertions';
+
+
+describe('browserDetector()', () => {
+  beforeEach(() => {
+    (globalThis.window as {}) = {};
+    sinon.stub(globalThis, 'window').value({
```

Comment:
The test is failing because `globalThis.window` on web environment is not configurable. You may need to stub the property you need instead of stub the whole "window" object.

Reply:
Test isn't failing...

Reply:
fixed. but it took me a while to realize we now have `describeNode` and `describeBrowser` 🤦